src/Microsoft.AspNetCore.Mvc.Analyzers/ApiControllerSymbolCache.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis;
+
+namespace Microsoft.AspNetCore.Mvc.Analyzers
+{
+    internal readonly struct ApiControllerSymbolCache
```

Comment:
Renamed `ApiControllerTypeCache` -> `ApiControllerSymbolCache`. We need to cache additional things like `IDisposable.Dispose` method symbol.